lib/exports/wrap-end.js

```diff
@@ -0,0 +1,18 @@
+	// CommonJS/npm, we want to export Box instead of assigning to global Window
```

Comment:
Since this and wrap-start are invalid js files, it might be better to have them issue of lib/ with a different extension.